src/originalBudgetTrackingApp/eventEmitters/consoleEmitter.ts

```diff
@@ -1,11 +1,9 @@
 /* eslint-disable class-methods-use-this */
-import { BudgetTrackingEventEmitter, EventPublisher } from '@/originalBudgetTrackingApp/eventEmitters/EventEmitter';
+import { EventNames, EventPublisher } from '@/originalBudgetTrackingApp/eventEmitters/EventEmitter';
 
-export function buildConsoleEmitter(): EventPublisher {
-  const consoleEmitter = new BudgetTrackingEventEmitter();
-  consoleEmitter.onAny((eventName, eventData) => {
+export default class ConsoleEmitter implements EventPublisher {
+  async emit(eventName: EventNames, eventData?: any) {
```

Comment:
This is strange to me. The emit should just publish the event, not handle it. The previous implementation created a generic emitter, and just implemented onAny that logged to the console. It handled the console log on the consumer side, which seems to me to make more sense.